test/unit/test_quilbase.py

```diff
@@ -152,6 +152,12 @@ def test_modifiers(self, gate: Gate):
     def test_repr(self, gate: Gate, snapshot: SnapshotAssertion):
         assert repr(gate) == snapshot
 
+    def test_eq(self, gate: Gate, name: str, params: List[ParameterDesignator], qubits: List[Qubit]):
+        assert gate == Gate(name, params, qubits)
```

Comment:
Optionally, just to round things out:

```suggestion
        assert gate == Gate(name, params, qubits)
        assert not gate != Gate(name, params, qubits)
```

Reply:
Good call!